app/common/src/Device.js

```diff
@@ -3,9 +3,16 @@
 import { Dimensions } from 'react-native';
 
 function renewDimensions() {
-  return Dimensions.get('screen');
+  try {
+    return Dimensions.get('screen');
+  } catch (e) {
+    return { width: 0, height: 0 };
+  }
```

Comment:
You are probably fixing something. So I miss tests...

Reply:
This actually fixes tests :) There is no `screen` when tests are run so it throws [this error](https://github.com/facebook/react-native/blob/0cd69e8a02d254577fac9ccd3ff8a1727a024cc8/Libraries/Utilities/Dimensions.js#L93).
I can put comment there or somehow mock globally.

Reply:
Ok, please add the comment explaining what is going on here.